indexing/src/test/java/com/google/enterprise/cloudsearch/sdk/indexing/StructuredDataTest.java

```diff
@@ -796,6 +760,28 @@ public void testDateTime_unparsedCharacters() throws IOException {
     converter.convert("2018-08-08T15:48:17.000-07:00 and so on");
   }
 
+  @Test
+  public void testIgnoreConversionErrors() throws IOException {
+    setupSchema();
+    StructuredData.setIgnoreConversionErrors(true);
+    StructuredDataObject expected =
+            new StructuredDataObject()
+                    .setProperties(
```

Comment:
the indentation looks different.  please use two spaces.
https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Reply:
IntelliJ was having trouble auto-detecting 2-spaces. I have become aware of the issue and will correct the code.

Reply:
Fixed and pushed